docs/getting_started/run_schedule.md

```diff
@@ -70,7 +79,10 @@ environment, you can import and execute it directly:
 
 from workflows.example import wf
 
-execution = remote.execute(wf, inputs={"name": "Kermit"})
+execution = remote.execute(wf,
+    inputs={"name": "Kermit"},
+    image_config=ImageConfig.auto(img_name="futureoutlier/flyte-practice:latest"),
```

Comment:
can we use `ghcr.io/flyteorg/flytekit:py3.11-1.9.1`

Reply:
No problem, I will test the image and paste the screenshot here!